docs/interfaces/postgres.rst

```diff
@@ -171,7 +171,8 @@ table available in CrateDB::
     |   21 | int2                         |     1005 |       0 |      2 | b       | N           |
     |   23 | int4                         |     1007 |       0 |      4 | b       | N           |
     |   24 | regproc                      |     1008 |       0 |      4 | b       | N           |
-    |   30 | oidvector                    |     1013 |       0 |     -1 | b       | A           |
+    |   26 | oid                          |     1013 |       0 |      4 | b       | N           |
```

Comment:
`oidvector` and `oid` have the same `typarray` oid. I've looked it up and it should be 1028 for `oid`:

```
template1=# select typarray from pg_type where typname = 'oid';
 typarray
----------
     1028
(1 row)
```

Reply:
Oh right, thanks for catching this :+1: